### PR TITLE
[DO NOT MERGE] Enable Fedora Rawhide

### PR DESCRIPTION
**Pull request by lsm5**
Signed-off-by: Lokesh Mandvekar <lsm5@fedoraproject.org>

### CODE REVIEW

File: base_images/fedora_base-setup.sh [hunk 1]
```diff
@@ -16,6 +16,11 @@ REPO_DIRPATH=$(realpath "$SCRIPT_DIRPATH/../")
 # shellcheck source=./lib.sh
 source "$REPO_DIRPATH/lib.sh"
 
+# Switch to Rawhide
```
Comment (lsm5) [inline base_images/fedora_base-setup.sh:19]: @cevich I see there's a `shellcheck disable=SC2154` here. Do I need to move it before this part ?

File: base_images/fedora_base-setup.sh [hunk 2]
```diff
@@ -16,6 +16,12 @@ REPO_DIRPATH=$(realpath "$SCRIPT_DIRPATH/../")
 # shellcheck source=./lib.sh
 source "$REPO_DIRPATH/lib.sh"
 
+# Update to Rawhide
+# NOTE: Should not break Fedora releases
+if [[ -z BUILD_RAWHIDE ]]; then
```
Comment (cevich) [inline base_images/fedora_base-setup.sh:21]: I think maybe you want a `-n` test here.

File: base_images/fedora_base-setup.sh (hunk 1 above)
Comment (cevich) [reply base_images/fedora_base-setup.sh]: You need to disable SC2154 for any env. var. that's defined elsewhere, like `$CIRRUS_foo` for example.

**Comment by cevich** [comment]
Taking this over as https://github.com/containers/automation_images/pull/273

**Pull request closed without merging**